pkg/codegen/docs/templates/function.tmpl

```diff
@@ -30,6 +30,9 @@
 <!-- Go -->
 {{ print "{{% choosable language go %}}" }}
 <div class="highlight"><pre class="chroma"><code class="language-go" data-lang="go"><span class="k">func </span>{{ .FunctionName.go }}<span class="p">(</span>{{ htmlSafe .FunctionArgs.go }}<span class="p">) (*{{ template "linkify_param" .FunctionResult.go }}, error)</span></code></pre></div>
+{{- if ne .FunctionName.go .Header.Title }}
+> This function's name was changed for the Go SDK to avoid conflicts with another similarly named function in the package.
```

Comment:
I could use some help with here with the wording. :)

Reply:
Perhaps something like the following?

```suggestion
> Note: This function is named `{{ .FunctionName.go }}` in the Go SDK.
```